### PR TITLE
Update grpc.md

### PR DESCRIPTION
**Pull request by vmw-yingy**

### CODE REVIEW

File: site/content/guides/grpc.md [hunk 1]
```diff
@@ -145,10 +145,44 @@ spec:
 
 ## Gateway API Configuration
 
-At the moment, configuring gRPC routes with Gateway API resources is achieved by the same method as with Ingress v1: annotation to select a protocol and port on a Service referenced by HTTPRoute `spec.rules[].backendRefs`.
+Gateway API now supports a specific resource [GRPCRoute][6] for routing gRPC requests.
```
Comment (skriss) [inline site/content/guides/grpc.md:148]: Please move this content into `site/content/docs/main/guides/grpc.md` instead, thanks!

File: site/content/guides/grpc.md [hunk 2]
```diff
@@ -145,10 +145,44 @@ spec:
...
+        service: yages.Echo
+        method: Ping
+    - method:
+        service: grpc.reflection.v1alpha.ServerReflection
+        method: ServerReflectionInfo
+    backendRefs:
+    - name: grpc-echo
+      port: 9000
+```
+Using the sample deployment above along with this GRPCRoute example, you can test calling this plaintext gRPC server with the same grpccurl command:
```
Comment (sunjayBhatia) [inline site/content/guides/grpc.md:175]: tiny nit
```suggestion
Using the sample deployment above along with this GRPCRoute example, you can test calling this plaintext gRPC server with the same grpcurl command:
```

File: site/content/guides/grpc.md (hunk 1 above)
Comment (vmw-yingy) [reply site/content/guides/grpc.md:148]: in main branch, there is no folder site/content/docs/main/guides/, shall I create one and git mv site/content/guides/grpc.md to there?

Comment (skriss) [reply site/content/guides/grpc.md:148]: https://github.com/projectcontour/contour/tree/main/site/content/docs/main/guides

File: site/content/docs/main/guides/grpc.md [hunk 3]
```diff
@@ -144,10 +144,44 @@ spec:
...
+        service: yages.Echo
+        method: Ping
+    - method:
+        service: grpc.reflection.v1alpha.ServerReflection
+        method: ServerReflectionInfo
+    backendRefs:
+    - name: grpc-echo
+      port: 9000
+```
+Using the sample deployment above along with this GRPCRoute example, you can test calling this plaintext gRPC server with the same grpccurl command:
```
Comment (sunjayBhatia) [inline site/content/docs/main/guides/grpc.md:174]:
```suggestion
Using the sample deployment above along with this GRPCRoute example, you can test calling this plaintext gRPC server with the same grpcurl command:
```

**Review by skriss — APPROVED** [review]
👍

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**